using-mobile-extensions/adobe-analytics-mobile-services/mobileservices-api-reference.md

```diff
@@ -72,169 +136,112 @@ Uri testUri = new Uri.Builder()
 
         MobileServices.trackAdobeDeepLink(testUri);
 ```
+
 {% endtab %}
 
-{% tab title="iOS — Obj-C" %}
-### trackAdobeDeepLink
+{% tab title="iOS (AEP 3.x)" %}
 
 **Syntax**
 
 ```objectivec
-+ (void) trackAdobeDeepLink: (NSURL*) url;
++ (void) trackAdobeDeepLink: (NSURL* _Nonnull) deeplink;
 ```
 
 **Example**
 
-**Objective C**
+**Objective-C**
 
 ```objectivec
 NSURL* url = [NSURL URLWithString:@"adobelinktest://x?a.deeplink.id=test_deeplinkId&a.launch.campaign.trackingcode=code&test_key=test_value"];
 
-[ACPMobileServices trackAdobeDeepLink:url];
+[AEPMobileServices trackAdobeDeepLink:url];
 ```
 
 **Swift**
 
 ```swift
 let url = URL(string: "adobelinktest://x?a.deeplink.id=test_deeplinkId&a.launch.campaign.trackingcode=code&test_key=test_value")!
-ACPMobileServices.trackAdobeDeepLink(url)
+AEPMobileServices.trackAdobeDeepLink(url)
 ```
+
 {% endtab %}
 
-{% tab title="iOS — Swift" %}
-### trackAdobeDeepLink
+{% tab title="iOS (ACP 2.x)" %}
 
 **Syntax**
 
 ```objectivec
-+ (void) trackAdobeDeepLink: (NSURL* _Nonnull) deeplink;
++ (void) trackAdobeDeepLink: (NSURL*) url;
 ```
 
 **Example**
 
-**Objective-C**
+**Objective C**
 
 ```objectivec
 NSURL* url = [NSURL URLWithString:@"adobelinktest://x?a.deeplink.id=test_deeplinkId&a.launch.campaign.trackingcode=code&test_key=test_value"];
 
-[AEPMobileServices trackAdobeDeepLink:url];
+[ACPMobileServices trackAdobeDeepLink:url];
 ```
 
 **Swift**
 
 ```swift
 let url = URL(string: "adobelinktest://x?a.deeplink.id=test_deeplinkId&a.launch.campaign.trackingcode=code&test_key=test_value")!
-AEPMobileServices.trackAdobeDeepLink(url)
+ACPMobileServices.trackAdobeDeepLink(url)
 ```
-{% endtab %}
-{% endtabs %}
-
-## Process referrer intent
 
-You can use this API to process the referrer intent that was received from Android.
+{% endtab %}
 
-{% hint style="warning" %}
-This API is **only** available in Android.
-{% endhint %}
+{% endtabs %}
 
-### processReferrer
+## extensionVersion
```

Comment:
Move `extensionVersion` to the top for alphabetical listing of APIs.

Reply:
https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-analytics/analytics-api-reference#extensionversion
We didn't do that for Analytics documentation, so I think we'd better not to change it.